chia/server/ws_connection.py

```diff
@@ -269,14 +270,16 @@ async def invoke(*args, **kwargs):
             if attribute is None:
                 raise AttributeError(f"Node type {self.connection_type} does not have method {attr_name}")
 
-            msg = Message(uint8(getattr(ProtocolMessageTypes, attr_name).value), None, args[0])
+            msg: Message = Message(uint8(getattr(ProtocolMessageTypes, attr_name).value), None, args[0])
             request_start_t = time.time()
             result = await self.create_request(msg, timeout)
             self.log.debug(
                 f"Time for request {attr_name}: {self.get_peer_logging()} = {time.time() - request_start_t}, "
                 f"None? {result is None}"
             )
             if result is not None:
+                if not sent_message_response_ok(msg.type, result.type):
+                    self.log(f"WSConnection.invoke sent message {msg.type} but received {result.type}")
```

Comment:
I would expect this to be a fatal error, at least fatal to the peer connection. This suggests that the peer is not speaking the protocol correctly, right? So I would think we'd want to throw an exception here, causing the connection to be closed.

Reply:
I see, right. Yes, I was aware it is a protocol violation, but I was thinking we'd attempt to keep going.

Hmm, I think you are right.